frontend/lib/requests/droplet.ts

```
@@ -139,3 +149,354 @@ export async function getRandomFunFactDroplet({
+}
+
+export async function updateDroplet(
+  id: number,
+  data: Partial<z.infer<typeof DropletSchema>>,
+  options: { regenerateSlug?: boolean; revalidate?: boolean } = {
+    regenerateSlug: false,
+    revalidate: false,
+  },
+) {
+  try {
+    const dataToSend: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)